shop_profile/tests.py

```
@@ -56,6 +68,10 @@ def test_shop_profile_str(self):
         """Test the string representation of ShopProfile."""
         self.assertEqual(str(self.shop), "Test Shop")
 
+    def test_shop_profile_check_password(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_shop_profile_check_password`

Add return type annotation: `None`

(ANN201)